app/client/lib/MultiUserManager.ts

```diff
@@ -0,0 +1,182 @@
+  model: UserManagerModel,
+  onAdd: (email: string, role: NonGuestRole) => void,
+) {
+  const emailListObs = Observable.create(owner, "");
+  const rolesObs = Observable.create<BasicRole>(owner, VIEWER);
+  const isValidObs = Observable.create(owner, true);
+  
+  const enableAdd: Computed<boolean> = computed((use) => Boolean(use(emailListObs) && use(rolesObs) && use(isValidObs)));
+  
+  const save = (ctl: IModalControl) => {
+    const emailList = parseEmailList(emailListObs.get())
+    const role = rolesObs.get();
+    if (emailList.some(email => !validateEmail(email))) {
+      isValidObs.set(false);
+    } else {
+      emailList.map(email => onAdd(email, role))
+      ctl.close()
+    }
+  }
+  const cssBody = model.isPersonal ? cssAccessDetailsBody : cssUserManagerBody;
+
+  return modal(ctl => [
+    // We set the padding to 0 since the body scroll shadows extend to the edge of the modal.
+    { style: 'padding: 0;' },
+    dom.cls(cssAnimatedModal.className),
+    cssTitle(
+      'Invite Team Members',
+      testId('um-header'),
+    ),
+    cssModalBody(
+      cssBody(
+        buildMultiUserManager(emailListObs, isValidObs),
+        dom.domComputed(isValidObs, isValid => !isValid ? cssErroMessage('At least one email is invalid') : null),
+        cssInheritRoles(
+          dom('span', 'Access: '),
+          buildRolesSelect(rolesObs, model)
+        )
+      ),
+    ),
+    cssModalButtons(
+      { style: 'margin: 32px 64px; display: flex;' },
+      (model.isPublicMember ? null :
+        bigPrimaryButton('Confirm',
+          dom.boolAttr('disabled', (use) => !use(enableAdd)),
+          dom.on('click', () => {save(ctl)}),
+          testId('um-confirm')
+        )
+      ),
+      bigBasicButton(
+        model.isPublicMember ? 'Close' : 'Cancel',
+        dom.on('click', () => ctl.close()),
+        testId('um-cancel')
+      ),
+    )
+  ])
+}
+
+function buildRolesSelect(
+  roleSelectedObs: Observable<BasicRole>,
+  model: UserManagerModel,
+) {
+  const allRoles = model.inheritSelectOptions
+    .filter((x): x is {value: BasicRole, label: string} => isBasicRole(x.value));
+  return cssOptionBtn(
+    menu(() => [
+      dom.forEach(allRoles, (_role) =>
+        menuItem(() => roleSelectedObs.set(_role.value), _role.label,
+          testId(`um-role-option`)
+        )
+      )
+    ]),
+    dom.text((use) => {
+      // Get the label of the active role.
+      const activeRole = allRoles.find((_role: IMemberSelectOption) => use(roleSelectedObs) === _role.value);
+      return activeRole ? activeRole.label : "";
+    }),
+    cssCollapseIcon('Collapse'),
+    testId('um-max-inherited-role')
+  );
+}
+
+
+function buildMultiUserManager(
+  emailListObs: Observable<string>,
+  isValidObs: Observable<boolean>,
+  // model: UserManagerModel,
+  ...args: DomElementArg[]
+) {
+  return cssTextarea(emailListObs,
+    {onInput: true, isValid: isValidObs},
+    {placeholder: "Enter one email address par mail"},
```

Comment:
```suggestion
    {placeholder: "Enter one email address per line"},
```

On a related note, is i18n for this module planned for a follow-up PR or should it be done now?